src/Standards/Generic/Tests/Formatting/MultipleStatementAlignmentUnitTest.js.fixed

```diff
@@ -83,7 +83,7 @@ var buttonWidget     = WidgetStore.get(buttonid);
 var spinButtonid     = [this.id, '-positionFormats-spinButton'].join('');
 var spinButtonWidget = WidgetStore.get(spinButtonid);
 var position         = spinButtonWidget.getValue();
-var posForamatsList  = WidgetStore.get([self.id, '-positionFormats-list'].join(''));
+var posFormatsList  = WidgetStore.get([self.id, '-positionFormats-list'].join(''));
```

Comment:
Needs an extra space after the label. You are breaking the test now.

Reply:
fixed